bazel/repositories.bzl

```diff
@@ -431,6 +444,26 @@ def _com_github_zlib_ng_zlib_ng():
         patches = ["@envoy//bazel/foreign_cc:zlib_ng.patch"],
     )
 
+def _org_boost():
+    external_http_archive(
+        name = "org_boost",
+        build_file_content = """
+filegroup(
+    name = "header",
+    srcs = glob([
+        "boost/**/*.h",
+        "boost/**/*.hpp",
+        "boost/**/*.ipp",
+    ]),
+    visibility = ["@envoy//contrib/hyperscan/matching/input_matchers/source:__pkg__"],
+)
+""",
+    )
+    native.bind(
```

Comment:
Do you need this target? I do not see it used anywhere.

Reply:
https://github.com/envoyproxy/envoy/pull/19473/files#diff-260492c6f1d705c11f7cb740768abd0aa639a568c250495d090fbaf67462e06fR25

Boost is only used as a dependency of Hyperscan. We do not actually use Boost to write any code in the contrib extension.

Reply:
I understand. I just do not see where the alias that native.bind is creating is used. What happens if you remove it?

Reply:
Yes, we can remove it, thanks.